server/graphql/v2/mutation/SendSurveyResponseMutation.ts

```diff
@@ -30,8 +30,8 @@ export const sendSurveyResponseMutation = {
     }
     const CODA_TOKEN = process.env.CODA_IN_APP_SURVEY_TOKEN;
 
-    if (!CODA_TOKEN) {
-      throw new UnexpectedError('Missing CODA_IN_APP_SURVEY_TOKEN');
+    if (!CODA_TOKEN && config.env !== 'development') {
+      throw new UnexpectedError('Missing CODA_IN_APP_SURVEY_TOKEN environment variable');
```

Comment:
I would argue for the opposite: never show such a technical error message to end-users.

```suggestion
    if (!CODA_TOKEN) {
      throw new UnexpectedError(config.env !== 'development' ? 'Missing CODA_IN_APP_SURVEY_TOKEN environment variable' : 'An unexpected error happened while submitting the survey');
```

Reply:
Shouldn't it be `(config.env === 'development' ? 'Missing CODA_IN_APP_SURVEY_TOKEN environment variable' : 'An unexpected error happened while submitting the survey')` instead?

Reply:
Indeed